client/core/controllers/apiController.h

```diff
@@ -20,7 +20,7 @@ public slots:
     void updateServerConfigFromApi(const QString &installationUuid, const int serverIndex, QJsonObject serverConfig);
 
 signals:
-    void errorOccurred(const QString &errorMessage);
+    void errorOccurred(ErrorCode errorCode);
```

Comment:
add const

Reply:
in private conversation decided that no sense to add here const
https://stackoverflow.com/questions/117293/does-using-const-on-function-parameters-have-any-effect-why-does-it-not-affect